api/features/migrations/0065_make_feature_value_size_configurable.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.25 on 2024-08-01 21:09
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('features', '0064_fix_feature_help_text_typo'),
+    ]
```

Comment:
Can we add the sqlmigrate for this and the other migration

Reply:
Here they are without setting the environment variable:

```
❯ python manage.py sqlmigrate features 0065
BEGIN;
--
-- Alter field initial_value on feature
--
-- (no-op)
--
-- Alter field initial_value on historicalfeature
--
-- (no-op)
--
-- Alter field string_value on featurestatevalue
--
-- (no-op)
--
-- Alter field string_value on historicalfeaturestatevalue
--
-- (no-op)
COMMIT;
❯ python manage.py sqlmigrate multivariate 0008
BEGIN;
--
-- Alter field string_value on historicalmultivariatefeatureoption
--
-- (no-op)
--
-- Alter field string_value on multivariatefeatureoption
--
-- (no-op)
COMMIT;
```

... and with the environment variable

```
❯ export FEATURE_VALUE_LIMIT=2000000
❯ python manage.py sqlmigrate features 0065
BEGIN;
--
-- Alter field initial_value on feature
--
ALTER TABLE "features_feature" ALTER COLUMN "initial_value" TYPE varchar(2000000);
--
-- Alter field initial_value on historicalfeature
--
ALTER TABLE "features_historicalfeature" ALTER COLUMN "initial_value" TYPE varchar(2000000);
--
-- Alter field string_value on featurestatevalue
--
ALTER TABLE "features_featurestatevalue" ALTER COLUMN "string_value" TYPE varchar(2000000);
--
-- Alter field string_value on historicalfeaturestatevalue
--
ALTER TABLE "features_historicalfeaturestatevalue" ALTER COLUMN "string_value" TYPE varchar(2000000);
COMMIT;
❯ python manage.py sqlmigrate multivariate 0008
BEGIN;
--
-- Alter field string_value on historicalmultivariatefeatureoption
--
ALTER TABLE "multivariate_historicalmultivariatefeatureoption" ALTER COLUMN "string_value" TYPE varchar(2000000);
--
-- Alter field string_value on multivariatefeatureoption
--
ALTER TABLE "multivariate_multivariatefeatureoption" ALTER COLUMN "string_value" TYPE varchar(2000000);
COMMIT;
```